MWChartsPlugin.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name                  = 'MWChartsPlugin'
-    s.version               = '1.0.7'
+    s.version               = '1.1.0'
```

Comment:
@PedroSebastiao Given that the minor version is being updated, this will require updating the App Builder portal once this version is released to the public.